src/effects/native/parametriceqeffect.cpp

```diff
@@ -0,0 +1,215 @@
+#include "effects/native/parametriceqeffect.h"
+#include "util/math.h"
+
+namespace {
+    constexpr int kBandCount = 2;
+    constexpr double kDefaultCenter1 = 1000; // 1 kHz
+    constexpr double kDefaultCenter2 = 3000; // 3 kHz
+}
+
+// static
+QString ParametricEQEffect::getId() {
+    return "org.mixxx.effects.parametriceq";
+}
+
+// static
+EffectManifest ParametricEQEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Parametric Equalizer"));
+    manifest.setShortName(QObject::tr("Param EQ"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+        "An gentle 2-band parametric equalizer based on biquad filters.\n"
+        "It is designed as a complement to the steep mixing EQs in the Effect rack\n"
+        "to improve the sound of the track."));
```

Comment:
"to improve the sound of the track" is so general that it could refer to any effect. Could we think of a better way to describe it? Alternatively, we may leave it as "It is designed as a complement to the steep mixing equalizers."